### PR TITLE
Automate - LPS-180813 | CP2FormForGuestUsers#AssertOnlyGuestUsersCanSeeTheForm

### PR DESCRIPTION
**Pull request by styller1**
Ticket:
[LPS-180812](https://liferay.atlassian.net/browse/LPS-180812)

### CODE REVIEW

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:Sf

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro [hunk 1]
```diff
@@ -180,6 +180,75 @@ definition {
...
+			value1 = "\ENTER");
+
+		Click.clickNoWaitForVisible(
+			key_fieldName = "acknowledgement",
+			locator1 = "Checkbox#CHECKBOX_NAME");
+
+		if (isSet(submit)) {
+			Button.click(button = "Submit");
+		}
+	}
```
Comment (luanakarla) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro:234]: Hey Pablo, 
could you turn the answers of the form into variables (and just add the answers in the macro call in the testcase)?

**Review by luanakarla — CHANGES_REQUESTED** [review]
Hey @styller1, thanks for working on it!
I requested a change in the macro `fulfillGuestForm`, could you please take a look at it?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro (hunk 1 above)
Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro:234]: Sure!

**Comment by styller1** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

**Review by luanakarla — APPROVED** [review]

**Comment by luanakarla** [comment]
The code looks good to me!
I'm forwarding it.

**Comment by luanakarla** [comment]
ci:forward

**Pull request closed without merging**